recipes/epics-base/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "3.14.12.7" %}
+{% set sha256 = "806c5288e9e6e192d04b39eacc71f60f11fe86cbe2712ce733dbf2b4da44cef0" %}
+
+package:
+  name: epics-base
+  version: {{ version }}
+
+source:
+  fn: baseR{{version}}.tar.gz
+  url: https://epics.anl.gov/download/base/baseR{{version}}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - config_site.diff  # [not win]
+    - config_common.diff  # [not win]
+    - rules_build.diff
+    - config_readline.diff  # [linux]
+
+build:
+  number: 1
+  features:
+    - vc9  # [win and py27]
+    - vc14  # [win and (py35 or py36)]
+
+requirements:
+  build:
```

Comment:
please use the new build/host syntax

Reply:
can you give us an example on how this should look like or where to look this up?

Reply:
https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml

Reply:
Hi @duncanmmacleod  finally somebody (i.e. @xiaoqiangwang) found some time to fix things in our recipes. However we cannot update this pull request as it is based on your fork, so please copy the new build.sh, bld.bat, meta.yaml into here and we guess things should be fine then.